examples/pytorch/diffpool/README.md

```diff
@@ -16,8 +16,8 @@ How to run
 ----------
 
 ```bash
-python train.py --dataset ENZYMES --pool_ratio 0.10 --num_pool 1
-python train.py --dataset DD --pool_ratio 0.15 --num_pool 1  
+python train.py --dataset ENZYMES --pool_ratio 0.10 --num_pool 1 --epochs 1000
```

Comment:
Can we merge the commands into a single run?

Reply:
Why? These two commands are for two different datasets.

Reply:
Oh I see, never mind.